test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsSolutionRestoreServiceTests.cs

```diff
@@ -2311,7 +2311,7 @@ public void ToPackageSpec_TargetFrameworkWithCLRSupport_InterpretsFrameworkCorre
             ProjectNames projectName = new ProjectNames(@"f:\project\project.vcxproj", "project", "project.csproj", "project", Guid.NewGuid().ToString());
             var emptyReferenceItems = Array.Empty<VsReferenceItem>();
             var packageReferenceProperties = new VsReferenceProperties();
-            var managedFramework = CommonFrameworks.Net50;
+            var managedFramework = NuGetFramework.Parse("net5.0-windows10.0");
```

Comment:
So is testing `.NETCoreApp,Version=5.0` now completely covered by `GetProjectFramework_WithCLRSupport_VariousInputs` , so that's why this version doesn't need to be tested in `ToPackageSpec_TargetFrameworkWithCLRSupport_InterpretsFrameworkCorrect`?

Reply:
I didn't work with @nkolev92 on this feature, but I think that C++/CLI projects will always advertise themselves as `netX.0-windows` because C++/CLI is windows-only. You can't run a C++/CLI assemblies can't run on Linux/Mac/other platforms that .NET runs on.  So, testing the platform agnostic TFM is a scenario that shouldn't be possible going forward, I think.